Sources/Brave/Frontend/Sync/BraveCore/BraveSyncAPIExtensions.swift

```diff
@@ -92,9 +92,13 @@ extension BraveSyncAPI {
     resetSync()
   }
 
-  func enableSyncTypes(syncProfileService: BraveSyncProfileServiceIOS) {
+  func enableSyncTypes(syncProfileService: BraveSyncProfileServiceIOS, shouldEnableBookmarks: Bool = true) {
```

Comment:
Same thing. You are already passing in explicit parameters anyway, so no need for these to be default values :)

Reply:
Fix at https://github.com/brave/brave-ios/pull/7918/commits/018fa5da8b358c050d1d9f7d00e454b06f316a75